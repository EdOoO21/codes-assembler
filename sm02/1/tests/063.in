-91 -398 905 862 -482 -9 351 -822 -209 535 -840 -846 350 -586 -405 -849 692 286 -372 725 528 251 -877 -545 355 730 928 784 -738 -328 -520 -440 226 379 -939 992 -932 869 855 795 222 465 -667 -534 562 255 137 -124 -117 308 -261 -3 433 -578 -784 838
