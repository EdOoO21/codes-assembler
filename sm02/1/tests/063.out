838
-784
-578
433
-3
-261
308
-117
-124
137
255
562
-534
-667
465
222
795
855
869
-932
992
-939
379
226
-440
-520
-328
-738
784
928
730
355
-545
-877
251
528
725
-372
286
692
-849
-405
-586
350
-846
-840
535
-209
-822
351
-9
-482
862
905
-398
-91
